swagger.yml

```diff
@@ -395,6 +395,32 @@ paths:
           description: List realm user filter, default to false. X-Total-Count header includes both enabled and disabled users, no matter if the enabled filter is set or not(incompatible).
           schema:
             type: boolean
+        - name: username
+          in: query
+          description: Filter by username
+          schema:
+            type: string
+        - name: fistName
+          in: query
+          description: Filter by fistName
+          schema:
+            type: string
+        - name: lastName
+          in: query
+          description: Filter by lastName
+          schema:
+            type: string
+        - name: email
+          in: query
+          description: Filter by email
+          schema:
+            type: string
+        - name: groupId
+          in: query
+          description: Filter by group
+          schema:
+            type: string
+            format: uuid
```

Comment:
Group ID sounds a bit troublesome when it comes to searching something. You would have to remember the UUID of the group in order to use this filter or process the search string in the UI first.

Reply:
Replaced with group name